server/db/connection.js

```diff
@@ -1,11 +1,17 @@
 const { Client } = require('pg');
-const dotenv = require('dotenv');
+require('dotenv').config();
```

Comment:
interesting, what's the difference between just `require('dotenv')` and `.config()`? I don't have the `.config()` in mine.